src/antsibull_nox/sessions/lint.py

```diff
@@ -648,11 +810,23 @@ def add_lint_sessions(
             run_black_modules=run_black_modules,
             black_config=black_config,
             black_package=black_package,
+            run_ruff_format=run_ruff_format,
+            ruff_format_config=ruff_format_config,
+            ruff_format_package=ruff_format_package,
+            run_ruff_autofix=run_ruff_autofix,
+            ruff_autofix_config=ruff_autofix_config,
+            ruff_autofix_package=ruff_autofix_package,
+            ruff_autofix_select=(
+                ["I"] if ruff_autofix_select is None else ruff_autofix_select
+            ),
```

Comment:
```suggestion
            ruff_autofix_select=ruff_autofix_select,
```

Reply:
This doesn't work, since `ruff_autofix_select` can be `None`, but the other function expects a list. `ruff_autofix_select or []` will work though.